spec/fixtures/module/preload-sandboxæø åü.js

```diff
@@ -0,0 +1,15 @@
+(function () {
+  const {setImmediate} = require('timers')
+  const {ipcRenderer} = require('electron')
+  window.ipcRenderer = ipcRenderer
+  window.setImmediate = setImmediate
+  window.require = require
+  if (location.protocol === 'file:') {
+    window.test = 'preload'
+    window.process = process
+  } else if (location.href !== 'about:blank') {
+    addEventListener('DOMContentLoaded', () => {
+      ipcRenderer.send('child-loaded', window.opener == null, document.body.innerHTML)
+    }, false)
+  }
+})()
```

Comment:
We are only testing for the successful load of this file with `window.test` variable, rest of the logic are irrelevant. Can you remove those ? Thanks!